### PR TITLE
Add Table of Contents to Console Docs

### PR DESCRIPTION
**Pull request by celanthe**
Began adding a table of contents (TOC) to console documentation, for ease of navigation to specific information in the documentation.

### CODE REVIEW

File: docs/home/console.md [hunk 1]
```diff
@@ -2,9 +2,81 @@
...
+15. [Selecting a Provisioning Method](#selecting-provisioning-methods)
+16. [Need Help Choosing?](#need-help-choosing)
+17. [How to Factory Reset Your Device](#factory-reset)
+18. [Android for Work Provisioning](#android-for-work)
+19. [Six Tap QR Code Provisioning](#six-tap-qr-code)
+20. [QR Code in ROM](#qr-code-in-ROM)
+21. [QR Code Reader Setup](#qr-code-reader-setup)
+22. [Handling Other Notifications During Provisioning](#handling-other-notifications)
+23. [IMEI/Serial-Number Based Provisioning](#imei-serial-provisioning)
+24. [Provisioning Templates](#provisioning-templates)
```
Comment (shivani61) [inline docs/home/console.md:29]: from here till line#39 all are sub-sections of template

File: docs/home/console.md [hunk 2]
```diff
@@ -2,9 +2,81 @@
...
+32. [Resetting to Default](#reset-to-default)
+33. [Apps Section](#apps-section)
+34. [Kiosk or Multi App Mode](#kiosk-multi-app-mode)
+35. [Advanced Settings](#advanced-settings)
+36. [Branding Section](#branding-section)
+37. [Settings Section](#settings-section)
+38. [Group Section](#group-section)
+39. [Preview Section](#preview-section)
+40. [Devices Section](#devices-section)
+41. [Getting Started](#getting-started)
```
Comment (shivani61) [inline docs/home/console.md:46]: all following are subsections till line 55

File: docs/home/console.md [hunk 3]
```diff
@@ -2,9 +2,81 @@
...
+47. [Settings Tab](#settings-tab)
+48. [Event Feed Tab](#event-feed-tab)
+49. [Device Graphs Tab](#device-graphs-tab)
+50. [Apps Tab](#apps-tab)
+51. [Compliance Policy Tab](#compliance-policy-tab)
+52. [Security Tab](#security-tab)
+53. [Remote Viewer Tab](#remote-viewer-tab)
+54. [Capture Logs Tab](#capture-logs-tab)
+55. [Connecting to a Device Using Secure Remote ADB](#connecting-using-remote-adb)
+56. [Groups](#groups)
```
Comment (shivani61) [inline docs/home/console.md:61]: similar feedback for groups, apps, alerts and other sections

**Comment by celanthe** [comment]
As this cleanup only related to readability on GitHub, I am closing this PR to direct my focus to where it is most needed.

**Pull request closed without merging**